apps/meteor/app/lib/server/functions/sendMessage.ts

```diff
@@ -292,6 +307,10 @@ export const sendMessage = async function (user: any, message: any, room: any, u
 	// TODO: is there an opportunity to send returned data to notifyOnMessageChange?
 	await afterSaveMessage(message, room);
 
+	if (uploadIdsToConfirm !== undefined) {
+		await Uploads.confirmTemporaryFiles(uploadIdsToConfirm, user._id);
```

Comment:
confirmTemporaryFiles only removes the file's `expiresAt`, while the code within the parser changes the file to `complete: true`, but the parser is not called for E2EE messages - are encrypted uploads being flagged as completed somewhere else?